src/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import './index.css';
+import Book from './pages/Book';
+import Categories from './pages/Categories';
+
```

Comment:
- [ ] Great job creating your bookstore app, it's looking great! Just a tip, usually this `index.js` file is named `App.js` when working with React.js, it's just a convention not mandatory.

Reply:
@smunozmo that is noted, thank you for that pointer!
Also great GIF! 🤣